core/src/main/scala/org/apache/spark/sql/execution/CachedPlanHelperExec.scala

```diff
@@ -116,7 +123,33 @@ object CachedPlanHelperExec extends Logging {
       newpls: mutable.ArrayBuffer[ParamLiteral]): Unit = {
     literals.foreach { case lv@LiteralValue(_, _, p) =>
       lv.value = newpls.find(_.pos == p).get.value
-      val y = newpls.find(_.pos == p).get.value
     }
   }
+
+  def setValue(dvd: DataValueDescriptor): Any = dvd match {
```

Comment:
Renamed to getValue. 

I looked into getObject but that would lead to unnecessary object creation, so in my opinion current implementation is preferable.